source-code/sdk-js/src/adapter-sveltekit/vite-plugin/transforms/refactoring.md

```diff
@@ -113,6 +113,144 @@ for markup block:
 
 # Flows
 
+## `Javascript` and `Svelte`
+
+There is some stuff that has to be done both in javascript AND every svelte file:
+
+Possible implementation:
+
+```ts
+function wrapAndAddParameters(
+	ast,
+	config,
+	wrappers: (
+		| "initHandleWrapper"
+		| "initRootLayoutServerLoadWrapper"
+		| "initServerLoadWrapper"
+		| "initActionWrapper"
+		| "initRequestHandlerWrapper"
+		| "initRootLayoutLoadWrapper"
+		| "initLoadWrapper"
+		| "initRootPageLoadWrapper"
+	)[],
```

Comment:
why is this an array?

Reply:
We can wrap multiple functions in one step.